Project.toml

```diff
@@ -1,4 +1,4 @@
-name = "BlockScalars"
+name = "MultilineStrings"
 uuid = "1e8d2bf6-9821-4900-9a2f-4d87552df2bd"
```

Comment:
Does this UUID not need to be updated as well?

```julia
Pkg.METADATA_compatible_uuid("MultilineStrings")
UUID("cd68b089-1eb1-539f-a380-f27e2a29ae22")
```

Reply:
This UUID was never  METADATA compatible. Also, this package has never been published so the UUID doesn't need to change.